pkg/controller/csr/csr_manager.go

```diff
@@ -38,6 +38,8 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	}
 
 	csrPredicateFuncs := predicate.Funcs{
+		GenericFunc: func(e event.GenericEvent) bool { return false },
```

Comment:
do we want to disable generic? (e.g. pod crashed/restarted when csr is created, so we missed the create event?)

Reply:
I think that edge case is ok, and this is same predicate as klusterlet-addon-controller 👍